src/api/index.ts

```diff
@@ -42,7 +43,7 @@ const posts = {
 }
 
 const post = {
-	get: (id?: string) => requests.get('post', id),
+	get: (id: string) => requests.get('posts', id),
```

Comment:
Config it so we can use the `/` here. it is more semantic
`get: (id: string) => requests.get('/posts', id),`